locales/fa/json.json

```diff
@@ -204,8 +204,8 @@
     "Done.": "انجام شده.",
     "Download": "دانلود",
     "Download Receipt": "دریافت رسید",
-    "Drop file or click to choose": "Drop file or click to choose",
-    "Drop files or click to choose": "Drop files or click to choose",
+    "Drop file or click to choose": "فایل را بکشید یا برای انتخاب کلیک کنید.",
```

Comment:
I think there is no need for a full stop at the end of the sentence. Or is it needed? What is correct in terms of grammar?